homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -71,10 +81,65 @@ def get_platforms(config_entry):
         for air_monitor_model in MODELS_AIR_MONITOR:
             if model.startswith(air_monitor_model):
                 return AIR_MONITOR_PLATFORMS
-
+    _LOGGER.error(
+        "Unsupported device found! Please create an issue at "
+        "https://github.com/syssi/xiaomi_airpurifier/issues "
+        "and provide the following data: %s",
+        model,
+    )
```

Comment:
I cannot commit to this branch directly, all PR's in this branch were already approved. Please create a new PR against the rewok branch to request the changes you want.

Reply:
> 
> 
> As the flow is not anymore related only to the airpurifier, the URL should probably not point to syssi's `xiaomi_airpurifier`?

Probably, so if you have a better idea, please submit so, because this URL was already here an I did not touch it, so it is not related to this PR.